string group_name
string[] names
string[] families
---
